src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -340,4 +364,23 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 		}
 		return undefined;
 	}
+
+	public fileToBase64 = (filepath, getType) => {
```

Comment:
This seems like a useful generic function to have - would suggest putting it in a more common location

Reply:
where do you think is a good place?

Reply:
Since this is encoding a file I'd suggest making a `src/sql/platform/files/common/files.ts` and putting the encode/decode functions there.